src/components/list-panel.tsx

```diff
@@ -57,7 +57,8 @@ const ListPanel: React.FunctionComponent<TodoItems> = ({ data, setData }) => {
   const toggleTask = async (task: Task) => {
     const currentIndex = data.indexOf(task)
     const newdata = [...data]
-    const newStatus = task.Status === Status.completed ? Status.active : Status.completed
+    const newStatus =
```

Comment:
```suggestion
    const newStatus = task.Status?.[0] === Status.completed ? [Status.active] : [Status.completed]
```
With typescript 3.7 you can write it like this.